biz.aQute.repository/src/aQute/bnd/repository/maven/provider/ReleaseDTO.java

```diff
@@ -16,7 +16,11 @@
 		EXPORT;
 	}
 
-	public static class JavadocDTO extends DTO {
+	public static class LocalReleaseDTO extends DTO {
```

Comment:
I am not sure this super type is wise. Both JavadocDTO and SourceDTO have `path` members, but we don't have supertypes to hold them.  I would rather just have JavadocDTO and SourceDTO each have their own `local` member.

Reply:
Done